### ISSUE TITLE
unclear usage of <prerelease> formatting

### ISSUE THREAD

**Issue by pniederlag** [ilovemilk/semver-git-plugin#25]
`showInfo` has a prerelease section. There is no clear documentation how to use/influence the output in this section. Reading sources and tests I am still not sure on how this is intendted to be used or wether it is really implemented yet. 

Other than that: thx for creating and sharing this really usefull tool!

**Comment by ilovemilk** [comment]
Hi Peter,
thanks for your PR and the issues.

The prerelease section refers to the [Semantic Versioning Documentation](https://semver.org/):

> A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphen [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92.

At the moment it's not possible to modify the prerelase version by some sort of incrementer  only be hand via the git tag. There was the idea to implement an incrementer for prereleases but I haven't found the time wrap my had around it.

I hope this helps! :)

**Comment by pniederlag** [comment]
Thx for your explanation. Now I got it. If there is a tag that has some -alpha, -beta, or -rc modifiers these tags are considered pre-release. That's pretty neat and I can think this is usefull for a bunch of people. :-> 
However I'd prefer to a prerelease not depend on a tag. Same reasoning as for not attaching SNAPSHOT to releaes/* branch. I'd like to use release/* branch as "prerelease-branch", producing ---rc release versions, maybe also -alpha, -beta.
So basically we end up in #27 again

**Comment by ilovemilk** [comment]
It would be possible to specify a formatter for a specific branch which adds a prereleae modifier e.g. develop -> -alpha modifier or release -> -rc.
Sadly, it's atm not possible to increment the prerelease modiflier e.g.: rc.1 to rc.2 and so on.
